packages/embed/src/utils/url-parameters-transfer.ts

```diff
@@ -0,0 +1,32 @@
+type Params = { [key: string]: string }
+
+type QueryStrings = {
+  [key: string]: any
+}
+
+const urlSearchToParams = (search: string) => {
+  var params: Params = {}
+  if (search !== '' && search !== null) {
+    var vars = search.split('&')
+    for (var i = 0; i < vars.length; i++) {
+      var pair = vars[i].split('=')
+      params[pair[0] as keyof Params] = decodeURIComponent(pair[1])
+    }
+  }
+  return params
+}
+
+export const transferUrlParametersToQueryStrings = (
```

Comment:
URL has just one query string. So you are not actually generating "query strings" but rather "query parameters". Please rename the method, my suggestion would be `getTransferableQueryParams`.

Please name the file the same as its main method, eg. if the method is `getTransferableQueryParams()` name the file `get-transferable-query-params.ts`.

Reply:
The file and all the functions were actually named like this, I simply imported the code and added ts

Reply:
We are trying to improve the lib, so we should rename things where appropriate.